appveyor.yml

```diff
@@ -1,9 +1,8 @@
-version: '0.3.0.{build}'
-assembly_info:
+version: '1.0.0.{build}'
```

Comment:
We aren't going from 0.3 to 1.0

Reply:
What should the version be? I was trying to follow [semver](http://semver.org/) to indicate a major change. However, since it is still targeting `net452`, maybe that isn't warranted. `0.4.0`?